### DISCUSSION TITLE
open pdf from command line

### DISCUSSION THREAD

**Discussion by goyalyashpal** [ahrm/sioyek#535] [Q&A] [answered]
* I was trying to following this guide [external-viewers] to add sioyek to its list, 
* ... it requires the command for the viewer. 
* i tried the exact command given there: `sioyek.exe /A page=%p "%1"` i.e. `sioyek.exe /A tutorial.pdf` didn't work
* i tried `sioyek.exe tutorial.pdf` (similar to `vim note.txt`) - only the blank sioyek screen opened
* i tried `sioyek --help` or `sioyek --version` - showed blank
* I tried reading the "[tutorial][815985d]", but
    * didn't find any section on launching with CLI or the likes. 
    * I was unable to dig out anything meaningfull from the section on Synctex - it doesn't show in the sioyek command where to specify the page number parameters, 
    * or even the [online sioyek manual] either.

So, can someone help 😅 

https://github.com/ahrm/sioyek/blob/7cb948face57b56ee55224b62d841c316e85294c/tutorial/tut.tex#L188
https://github.com/ahrm/sioyek/blob/7cb948face57b56ee55224b62d841c316e85294c/tutorial/tut.tex#L203

[external-viewers]: https://www.sumatrapdfreader.org/docs/Customize-external-viewers
[815985d]: https://github.com/ahrm/sioyek/blob/7cb948face57b56ee55224b62d841c316e85294c/tutorial/tut.tex#L176-L199
[online sioyek manual]: https://sioyek-documentation.readthedocs.io/en/latest/commands.html

### Version:
* sioyek: i couldn't find any about/help/info/version command in the command palette, and as mentioned above, couldn't figure out the CLI flags either
* i checked via hovering over the executable file itself, and it's 1.5.0

**Comment by ahrm** [comment, marked as answer]
> i tried the exact command given there: sioyek.exe /A page=%p "%1" i.e. sioyek.exe /A tutorial.pdf didn't work

Well, these are the options for foxitreader, they don't work for sioyek, probably something like this works:
```
sioyek.exe "%1" --page %p
```

**Reply by GitHubRulesOK** [reply]
call exe with -h to see cli options

**Reply by goyalyashpal** [reply]
> call exe with -h to see cli options

wow, strange, it doesn't work with msys2's `bash` but worked with `cmd` shell
